data/common.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
instead of putting the common data into hiera, can you please put them into the init.pp? That enabled puppet-string to get the default data and write it into the REREFENCE.md

Reply:
Done in https://github.com/voxpupuli/puppet-fail2ban/pull/135/commits/1ceb3ff9bcdf4eb18ab4b1512912db63230f62c5